core/net6.0/Dockerfile

```diff
@@ -37,6 +37,6 @@ RUN apk update \
 
 WORKDIR /app
 COPY --from=build /app/Apache.OpenWhisk.Runtime.Dotnet.Minimal/out ./
-ENV ASPNETCORE_URLS http://+:8080
+ENV ASPNETCORE_URLS=http://+:8080
```

Comment:
I forgot to mention that this is getting rid of a deprecation warning while building the image.